rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/BytemanHelper.java

```diff
@@ -0,0 +1,25 @@
+package io.narayana.lra.coordinator.domain.model;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class BytemanHelper {
+    static AtomicBoolean businessCalled = new AtomicBoolean(false);
+    public void rendezvousWait() throws InterruptedException {
+        synchronized (businessCalled) {
+            businessCalled.set(true);
+            businessCalled.notifyAll();
+            businessCalled.wait();
+        }
+    }
+    public void rendezvousNotify() throws InterruptedException {
+        synchronized (businessCalled) {
+            while (!businessCalled.get()) {
+                businessCalled.wait();
+            }
+            businessCalled.notify();
+        }
+    }
```

Comment:
I don't think your test uses these parts of the BytemanHelper?

Reply:
I can remove them.

Reply:
Thank you